eng/pipelines/stress/http-linux.yml

```diff
@@ -6,7 +6,7 @@ pool:
 variables:
   httpStressProject: src/System.Net.Http/tests/StressTests/HttpStress
   sdkBaseImage: sdk-corefx-current
-  httpStressImage: httpstress
+  HTTPSTRESS_IMAGE: httpstress
```

Comment:
I don't understand YAML sufficiently. But why is this in all capital letters but the other tags ('sdkBaseImage') not?  Why is that a change for the better?

Reply:
That's a good point. I guess it reflects the fact that the particular variable is getting consumed by docker-compose (which uses all-caps variable names). The rest just follow convention used in other pipelines in this repo. I don't have strong opinions on which is preferable.

For the sake of consistency I could maybe declare the variable using camelCase then map to the all caps name for the docker-compose task. @safern opinions?

Reply:
I think we should try and be consistent within the yaml file, if you want to use upper case, then let's make all the variables upper case, but I think mapping them and having them as camelCase make sense.